definitions/firefox_ios.toml

```diff
@@ -260,6 +260,77 @@ friendly_name = "Clicked at least one CTA"
 description = "This metric looks at proportion of all new profiles that were exposed to onboarding cards and clicked at least one action during on-boarding."
 owner = "rbaffourawuah@mozilla.com"
 
+[metrics.impressions]
+data_source = "appstore_funnel"
+select_expression = "SUM(impressions)"
+type = "scalar"
+friendly_name = "Firefox iOS appstore impressions"
+description = """
+    This is the number of unique impressions of firefox browser in iOS appstore. The etl of the base table is
+    [defined in `bigquery-etl`](https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_funnel_v1/query.sql).
+    This metric needs to be aggregated by `first_seen_date` (date column from the data recieved from appstore) for daily aggregation. The underlying table have a lag of 7 days.
+    For questions please contact "rbaffourawuah@mozilla.com".
+"""
+owner = "rbaffourawuah@mozilla.com"
+deprecated = false
+
+[metrics.downloads]
+data_source = "appstore_funnel"
+select_expression = "SUM(total_downloads)"
+type = "scalar"
+friendly_name = "Firefox iOS appstore downloads"
+description = """
+    This is the total number of downloads of firefox browser in iOS appstore. The etl of the base table is
+    [defined in `bigquery-etl`](https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_funnel_v1/query.sql).
+    This metric needs to be aggregated by `first_seen_date` (date column from the data recieved from appstore) for daily aggregation. The underlying table have a lag of 7 days.
+    For questions please contact "rbaffourawuah@mozilla.com".
+"""
+owner = "rbaffourawuah@mozilla.com"
+deprecated = false
+
+
+[metrics.new_profiles]
```

Comment:
My suggestions is to call this funnel_new_profiles, to avoid confusion with what we normally call new_profiles, which even though isn't defined in metric hub yet, it should have been, pointing to the firefox_ios_clients table.

This is particularly important once we create the same metrics on Android, which we want to keep consistent, as the Android funnel new profiles is a very small subset of total new profiles.

The same comment is applicable to the other retention metrics

Reply:
Updated the new_profiles name